crypto/bio/bf_lbuf.c

```diff
@@ -228,6 +239,25 @@ static long linebuffer_ctrl(BIO *b, int cmd, long num, void *ptr)
             ret = BIO_ctrl(b->next_bio, cmd, num, ptr);
         }
         break;
+    case BIO_C_SET_PREFIX:
+        {
+            size_t strl = strlen((const char *)ptr);
+            size_t saved = ctx->obuf_len - ctx->prefix_len;
+
+            if (saved > LINEBUFFER_PREFIX_MAXSIZE) {
```

Comment:
Why do we do this?

Reply:
As opposed to...?

Reply:
Lets say we have 100 bytes pending in `obuf` and the old prefix was 5 in length. That makes `saved` have the value 95. According to this code that results an error (95 > 40) - but I'm not sure why?